docs/client_libraries/go.mdx

```diff
@@ -43,210 +43,13 @@ go mod tidy
 ## Quick Start
 
 ### Push files to a remote repository
-
-```go
-package main
-
-import (
-    "context"
-    "fmt"
-
-    v1 "github.com/opencontainers/image-spec/specs-go/v1"
-    "oras.land/oras-go/v2"
-    "oras.land/oras-go/v2/content/file"
-    "oras.land/oras-go/v2/registry/remote"
-    "oras.land/oras-go/v2/registry/remote/auth"
-    "oras.land/oras-go/v2/registry/remote/retry"
-)
-
-func pushFiles() error {
-    // 0. Create a file store
-    fs, err := file.New("/tmp/")
-    if err != nil {
-        return err
-    }
-    defer fs.Close()
-    ctx := context.Background()
-
-    // 1. Add files to a file store
-    mediaType := "example/file"
-    fileNames := []string{"/tmp/myfile"}
-    fileDescriptors := make([]v1.Descriptor, 0, len(fileNames))
-    for _, name := range fileNames {
-        fileDescriptor, err := fs.Add(ctx, name, mediaType, "")
-        if err != nil {
-            return err
-        }
-        fileDescriptors = append(fileDescriptors, fileDescriptor)
-        fmt.Printf("file descriptor for %s: %v\n", name, fileDescriptor)
-    }
-
-    // 2. Pack the files and tag the packed manifest
-    artifactType := "example/files"
-    manifestDescriptor, err := oras.Pack(ctx, fs, artifactType, fileDescriptors, oras.PackOptions{
-        PackImageManifest: true,
-    })
-    if err != nil {
-        return err
-    }
-    fmt.Println("manifest descriptor:", manifestDescriptor)
-
-    tag := "latest"
-    if err = fs.Tag(ctx, manifestDescriptor, tag); err != nil {
-        return err
-    }
-
-    // 3. Connect to a remote repository
-    reg := "myregistry.example.com"
-    repo, err := remote.NewRepository(reg + "/myrepo")
-    if err != nil {
-        panic(err)
-    }
-    // Note: The below code can be omitted if authentication is not required
-    repo.Client = &auth.Client{
-        Client: retry.DefaultClient,
-        Cache:  auth.DefaultCache,
-        Credential: auth.StaticCredential(reg, auth.Credential{
-            Username: "username",
-            Password: "password",
-        }),
-    }
-
-    // 3. Copy from the file store to the remote repository
-    _, err = oras.Copy(ctx, fs, tag, repo, tag, oras.DefaultCopyOptions)
-    return err
-}
-
-func main() {
-    if err := pushFiles(); err != nil {
-        panic(err)
-    }
-}
-```
+See [example](https://pkg.go.dev/oras.land/oras-go/v2#example-package-PushFilesToRemoteRepository).

```

Comment:
i think it's better to say "See this example" no? But else LGTM

Reply:
Might be better just to change this to a bullet list, but this is good enough for now